homeassistant/components/zwave_js/cover.py

```diff
@@ -165,7 +165,13 @@ async def async_stop_cover(self, **kwargs: Any) -> None:
             or self.get_zwave_value(COVER_UP_PROPERTY)
             or self.get_zwave_value(COVER_ON_PROPERTY)
         )
-        if open_value:
+
+        if not open_value.metadata.readable:
```

Comment:
What is the purpose of adding this check? All of these switch type properties, Up, Down, Close, Open, [are write-only](https://zwave-js.github.io/node-zwave-js/#/api/CCs/MultilevelSwitch?id=levelchangedownswitchtype-switchtype).

Reply:
Thanks for the Z-Wave reference, in reviewing a prior fixture the state is mocked up as readable. https://github.com/home-assistant/core/blob/3a8282d0c5f7c2f72b677e84395543797418ba1a/tests/components/zwave_js/fixtures/cover_aeotec_nano_shutter_state.json#L214-L218

This may not actually be the case in the real work if your reference is the sole truth and the mock is incorrect.
@p4p3r  Do you have any experience with the Nano Shutter `Open`/`Close` state actually being readable during https://github.com/home-assistant/core/pull/52881 testing?

Reply:
The mock files do get out of date as the driver changes. The up/down values were made write-only in [this change](https://github.com/zwave-js/node-zwave-js/commit/603d27cf4450865ae1c3026b8fa6742ce551d62c).

Regardless, the read/write state of the value ID doesn't have anything to do with the functionality of the device. That is metadata for the value ID.